apps/crm/crm-e2e/src/integration/create-org.spec.ts

```diff
@@ -0,0 +1,205 @@
+import {
+  App,
+  canAccessModule,
+  createModuleAccess,
+  getOrgAppAccess,
+  getOrgModuleAccess,
+  orgActivity,
+  Organization,
+  organizationStatus,
+  territories,
+  User,
+  Module,
+} from '@blockframes/model';
+import {
+  // plugins
+  adminAuth,
+  firestore,
+  maintenance,
+  // cypress commands
+  get,
+  connectOtherUser,
+  assertUrl,
+  interceptEmail,
+  deleteEmail,
+} from '@blockframes/testing/cypress/browser';
+import { admin, Newcomer, newcomers } from '../fixtures/create-org';
+
+const injectedData = {
+  [`users/${admin.user.uid}`]: admin.user,
+  [`blockframesAdmin/${admin.user.uid}`]: {},
+  [`orgs/${admin.org.id}`]: admin.org,
+  [`permissions/${admin.permissions.id}`]: admin.permissions,
+};
+
+describe('Create organization', () => {
+  before(() => {
+    cy.visit('');
+    maintenance.start();
+    firestore.clearTestData();
+    adminAuth.deleteAllTestUsers();
+    firestore.create([injectedData]);
+    adminAuth.createUser({ uid: admin.user.uid, email: admin.user.email, emailVerified: true });
+    maintenance.end();
+    cy.visit('');
+    connectOtherUser(admin.user.email);
+  });
+
+  for (const newcomer of newcomers) {
+    const { org } = newcomer;
+    const apps = getOrgAppAccess(org);
+    const modules = getOrgModuleAccess(org);
+    it(`Create ${org.status} org on ${apps.join(' & ')} which can access ${modules.join(' & ')}`, () => {
+      firestore.queryDeleteOrgsWithUsers({
+        collection: 'orgs',
+        field: 'addresses.main.street',
+        operator: '==',
+        value: org.addresses.main.street,
+      });
+      createOrg(newcomer);
+      checkEmail(newcomer);
+      checkDbDocs(newcomer);
+      checkUI(newcomer);
+    });
+  }
+});
+
+//* functions
+
+function createOrg(newcomer: Newcomer) {
+  const { org, user } = newcomer;
+  const apps = getOrgAppAccess(org);
+  const modules = getOrgModuleAccess(org);
+  const fromApp = apps.includes('festival') ? 'festival' : 'catalog';
```

Comment:
you can use
```suggestion
  const fromApp = apps[0];
```
as  `getOrgAppAccess` will put `festival` as first array value if org have access to it